eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -284,7 +284,7 @@ stages:
       dependsOn: Signing
       jobs:
       - job: PublishPackages
-        condition: or(eq(variables['SetDevVersion'], 'true'), and(eq(variables['Build.Reason'],'Schedule'), eq(variables['System.TeamProject'], 'internal')))
+        condition: and(eq(variables['SetDevVersion'], 'true'), eq(variables['System.TeamProject'], 'internal'))
```

Comment:
Based on our recent issue in JS I assume this PR needs to be revisited.

Reply:
Yep.